doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1388,19 +1388,15 @@ Response:
 
 This operation returns the `value` property with the value of the attribute.
 
-* If the response payload MIME type is `application/json`:
-  * If the attribute value is a JSON object or array, the payload is that JSON object or array
-  * If the attribute value is a string, a number, null or a boolean, an HTTP error 
-    "406 Not Acceptable: accepted MIME types: text/plain" is returned.
-* If the response payload MIME type is `text/plain`,
-  * If the attribute value is a JSON object or array, the payload in a stringification of that JSON
-    object or array.
-  * If the attribute value is a string, number, null or boolean, the payload is a text
-    representation of that value. In case of a string, citation marks are used at the begining and
-    end.
-
-The payload MIME type in responses is based on client-server HTTP negotiation, based on the client
-`Accept` header.
+* If attribute value is JSON Array or Object:
```

Comment:
Regarding using "expansion" rather than "contain" when referring to Accept, fixed in ae27c94c17e3be70c9ae8a3d60b65cffd793e484.

Regarding changing the orientation in the description, NTC, as I think is more clear to describe from the perspective of the attribute value (first level bullets), then Accept (second level bullets), as #1849 suggest.